rewrite-java-17/src/main/java/org/openrewrite/java/isolated/ReloadableJava17ParserVisitor.java

```diff
@@ -1472,10 +1472,14 @@ private J.VariableDeclarations visitVariables(List<VariableTree> nodes, Space fm
 
         Space varargs = null;
         if (typeExpr != null && typeExpr.getMarkers().findFirst(JavaVarKeyword.class).isEmpty()) {
-            int varargStart = indexOfNextNonWhitespace(vartype.getStartPosition(), source);
+            int varargStart = indexOfNextNonWhitespace(vartype.pos, source);
```

Comment:
`DiagnosticPosition.getPreferredPosition()` returns the `pos` field. Use that since the interface is less likely to change in future java versions than the field name.